packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barriers-list.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const BarriersList = ({
+    active_item_class_name,
+    base_class_name,
+    chosen_item,
+    className,
+    hover_item_class_name,
+    list,
+    onClick,
+    onMouseEnter,
+    onMouseLeave,
+}) => {
+    const barriers_list = list.map(barrier => {
+        const is_active = chosen_item === barrier;
```

Comment:
@kate-deriv it seems this variable is used only once, please use `selected_item === barrier` directly🙂

Reply:
Done, thank you